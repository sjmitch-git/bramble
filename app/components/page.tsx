import Link from 'next/link'

export default function Components() {
	return (
		<>
			<h1>Components</h1>
			<nav>
				<ul>
					<li>
						<Link href='./components/accordian'>Accordian</Link>
					</li>
					<li>
						<Link href='./components/alert'>Alert</Link>
					</li>
					<li>
						<Link href='./components/badge'>Badge</Link>
					</li>
					<li>
						<Link href='./components/breadcrumbs'>Breadcrumbs</Link>
					</li>
					<li>
						<Link href='./components/buttons'>Buttons</Link>
					</li>
					<li>
						<Link href='./components/buttongroup'>Button Group</Link>
					</li>
					<li>
						<Link href='./components/codeblock'>Codeblock</Link>
					</li>
					<li>
						<Link href='./components/figure'>Figure</Link>
					</li>
					<li>
						<Link href='./components/gallery'>Gallery</Link>
					</li>
					<li>
						<Link href='./components/modal'>Modal</Link>
					</li>
					<li>
						<Link href='./components/pagination'>Pagination</Link>
					</li>
					<li>
						<Link href='./components/ratings'>Ratings</Link>
					</li>
					<li>
						<Link href='./components/select'>Select</Link>
					</li>
					<li>
						<Link href='./components/video'>Video</Link>
					</li>
				</ul>
			</nav>
		</>
	)
}
