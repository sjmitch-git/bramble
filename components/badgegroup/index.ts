export * from './Badgegroup'
export { default } from './Badgegroup'
