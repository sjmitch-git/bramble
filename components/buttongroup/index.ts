export * from './Buttongroup'
export { default } from './Buttongroup'
