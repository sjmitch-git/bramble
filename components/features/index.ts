export * from './forms'
export * from './cards'
export * from './graphql'
export * from './navbar'
export * from './speech'
