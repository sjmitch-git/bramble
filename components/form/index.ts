export * from './Form'
export * from './NewsLetter'
export * from './LogIn'
export * from './Register'
export * from './User'
export * from './CreditCard'
