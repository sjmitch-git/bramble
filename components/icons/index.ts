export * from './DevIcon'
export * from './FacebookIcon'
export * from './GithubIcon'
export * from './InstagramIcon'
export * from './LinkedinIcon'
export * from './TwitterIcon'
export * from './XIcon'
export * from './YoutubeIcon'
