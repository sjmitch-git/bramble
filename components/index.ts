export * from './alert'

export * from './accordian'
export { default } from './accordian'

export { Autocomplete } from './autocomplete'
export * from './badge'
export * from './badgegroup'
export * from './brand'
export * from './breadcrumbs'
export * from './button'
export * from './buttongroup'
export * from './card'
export * from './carousel'
export * from './codeblock'
export * from './dialog'
export * from './dropdown'
export * from './figure'
export * from './footer'
export * from './form'
export * from './gallery'
export * from './icons'
export { Input } from './input'
export * from './loading'
export * from './modal'
export * from './navbar'
export * from './pagination'
export * from './range'
export * from './ratings'
export * from './select'
export * from './spinners'
export * from './switch'
export * from './table'
export * from './video'
export * from './videoplayer'
