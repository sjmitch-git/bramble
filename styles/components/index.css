@import './accordion.css';
@import './alert.css';
@import './badge.css';
@import './brand.css';
@import './breadcrumbs.css';
@import './buttons.css';
@import './buttongroup.css';
@import './card.css';
@import './carousel.css';
@import './counter.css';
@import './dialog.css';
@import './drawer.css';
@import './dropdown.css';
@import './figure.css';
@import './form.css';
@import './footer.css';
@import './gallery.css';
@import './input.css';
@import './label.css';
@import './loading.css';
@import './modal.css';
@import './navbar.css';
@import './pagination.css';
@import './radiogroup.css';
@import './range.css';
@import './ratings.css';
@import './select.css';
@import './sidebar.css';
@import './spinner.css';
@import './switch.css';
@import './tabs.css';
@import './toast.css';
@import './video.css';
@import './videoplayer.css';
