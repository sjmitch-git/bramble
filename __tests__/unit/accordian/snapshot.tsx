import { render } from '@testing-library/react'
import { Accordian } from '@/components'

import data from '@/data/users.json'

it('renders Accordian unchanged', () => {
	const { container } = render(<Accordian data={data} />)
	expect(container).toMatchSnapshot()
})
