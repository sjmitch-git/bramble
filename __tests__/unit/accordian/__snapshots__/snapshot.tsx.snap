// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Accordian unchanged 1`] = `
<div>
  <div
    class="accordian  "
    data-testid="accordian"
  >
    <div
      class="accordian-container "
    >
      <h3
        class="accordion-header "
      >
        Jeanette Penddreth
        <button
          class="btn  circle icon !static bg-light text-3xl text-primary"
          role="button"
          type="button"
        >
          <svg
            aria-hidden="true"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12 3.75a.75.75 0 01.75.75v6.75h6.75a.75.75 0 010 1.5h-6.75v6.75a.75.75 0 01-1.5 0v-6.75H4.5a.75.75 0 010-1.5h6.75V4.5a.75.75 0 01.75-.75z"
              fill-rule="evenodd"
            />
          </svg>
          <span
            class="sr-only"
          >
            Toggle section
          </span>
        </button>
      </h3>
      <section
        class="accordion-content"
        role="section"
      >
        <div>
          <p>
            <strong>
              Ugh umami
            </strong>
             butcher 
            <a
              href="#"
            >
              pickled kogi
            </a>
             stumptown, 
            <em>
              subway tile
            </em>
             gatekeep flexitarian. Bespoke meggings health goth meh ascot listicle, cronut vaporware. Marfa church-key cred, kinfolk mlkshk tousled roof party XOXO readymade solarpunk tbh adaptogen. Raclette williamsburg chicharrones, +1 retro hashtag freegan sustainable tumblr polaroid deep v tote bag godard.
          </p>
        </div>
      </section>
    </div>
    <div
      class="accordian-container "
    >
      <h3
        class="accordion-header "
      >
        Giavani Frediani
        <button
          class="btn  circle icon !static bg-light text-3xl text-primary"
          role="button"
          type="button"
        >
          <svg
            aria-hidden="true"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12 3.75a.75.75 0 01.75.75v6.75h6.75a.75.75 0 010 1.5h-6.75v6.75a.75.75 0 01-1.5 0v-6.75H4.5a.75.75 0 010-1.5h6.75V4.5a.75.75 0 01.75-.75z"
              fill-rule="evenodd"
            />
          </svg>
          <span
            class="sr-only"
          >
            Toggle section
          </span>
        </button>
      </h3>
      <section
        class="accordion-content"
        role="section"
      >
        <div>
          <p>
            <strong>
              Technologies
            </strong>
          </p>
          <ul
            class="list-disc list-inside"
          >
            <li>
              Blue bottle
            </li>
            <li>
              Sriracha heirloom distillery
            </li>
            <li>
              Offal roof party
            </li>
          </ul>
        </div>
      </section>
    </div>
    <div
      class="accordian-container "
    >
      <h3
        class="accordion-header "
      >
        Noell Bea
        <button
          class="btn  circle icon !static bg-light text-3xl text-primary"
          role="button"
          type="button"
        >
          <svg
            aria-hidden="true"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12 3.75a.75.75 0 01.75.75v6.75h6.75a.75.75 0 010 1.5h-6.75v6.75a.75.75 0 01-1.5 0v-6.75H4.5a.75.75 0 010-1.5h6.75V4.5a.75.75 0 01.75-.75z"
              fill-rule="evenodd"
            />
          </svg>
          <span
            class="sr-only"
          >
            Toggle section
          </span>
        </button>
      </h3>
      <section
        class="accordion-content"
        role="section"
      >
        <div>
          <p>
            Meh vice gatekeep flexitarian banjo lo-fi. Chicharrones mixtape 8-bit, iceland intelligentsia mlkshk direct trade pug.
          </p>
           
          <p>
            Pabst microdosing yes plz williamsburg, blog taxidermy put a bird on it ramps lyft. Next level meditation pok pok fashion axe JOMO scenester.
          </p>
        </div>
      </section>
    </div>
    <div
      class="accordian-container "
    >
      <h3
        class="accordion-header "
      >
        Willard Valek
        <button
          class="btn  circle icon !static bg-light text-3xl text-primary"
          role="button"
          type="button"
        >
          <svg
            aria-hidden="true"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M12 3.75a.75.75 0 01.75.75v6.75h6.75a.75.75 0 010 1.5h-6.75v6.75a.75.75 0 01-1.5 0v-6.75H4.5a.75.75 0 010-1.5h6.75V4.5a.75.75 0 01.75-.75z"
              fill-rule="evenodd"
            />
          </svg>
          <span
            class="sr-only"
          >
            Toggle section
          </span>
        </button>
      </h3>
      <section
        class="accordion-content"
        role="section"
      >
        <div>
          <p>
            Copper mug fam hexagon street art mixtape tumblr occupy ramps slow-carb air plant. Truffaut vegan cupping migas. Humblebrag bruh viral tumblr, whatever 90's butcher post-ironic meditation street art copper mug flannel.
          </p>
          <a
            class="btn mx-auto"
            href="#"
          >
            More
          </a>
        </div>
      </section>
    </div>
  </div>
</div>
`;
